Sources/Templates/Template.c

```diff
@@ -4,7 +4,7 @@
 #include <stdlib.h>
 #include <assert.h>
 
-void Template_AddMinuitia(Template *template, TemplateMinutia *minutia)
```

Comment:
I have no idea, we need to decide on a few things like that and then apply them to all the code in another branch I think...